apps/bfd-data-fda/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {
+        kubernetes {
+            yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  serviceAccount: bfd
+  containers:
+  - name: bfd-cbc-build
+    image: public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk11-mvn3-an29-tfenv
```

Comment:
Addition of that TODO is still warranted here. Can you re-add the TODO on resolving the container image rather than hard-coding it?

Reply:
done